src/index.html

```diff
@@ -7,8 +7,71 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymus"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo_link">
+        <img
+          src="/images/logo.png"
+          alt="Logo of company"
+          class="logo_img"
+        >
+      </a>
+      <nav class="nav_list">
+        <ul class="ul_list nav_list">
+          <li class="li_item nav_list">
+            <a href="#Apple" class="links_style is-active nav_list">
+              Apple
+            </a>
+          </li>
+          <li class="li_item nav_list">
+            <a href="#Samsung" class="links_style nav_list">
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
      <a href="#home" class="logo_link">
        <img
          src="/images/logo.png"
          alt="Logo of company"
          class="logo_img"
        >
      </a>
      
      <nav class="nav_list">
        <ul class="ul_list nav_list">
          <li class="li_item nav_list">
            <a href="#Apple" class="links_style is-active nav_list">
              Apple
            </a>
          </li>
          
          <li class="li_item nav_list">
            <a href="#Samsung" class="links_style nav_list">
              Samsung
            </a>
          </li>
          
```
add blanks between multiline elements

Reply:
check all such cases